shared-bindings/_pixelbuf/__init__.c

```diff
@@ -41,40 +41,9 @@
 //| Byteorders are configured with strings, such as "RGB" or "RGBD"."""
 // TODO: Pull in docs from pypixelbuf.
 
-//| def colorwheel(n: float) -> int:
-//|     """C implementation of the common wheel() function found in many examples.
-//|     Returns the colorwheel RGB value as an integer value for n (usable in :py:class:`PixelBuf`, neopixel, and dotstar)."""
-//|     ...
-//|
-//| def wheel(n: float) -> int:
-//|     """Use of wheel() is deprecated. Please use colorwheel()."""
-//|
-
-STATIC mp_obj_t pixelbuf_colorwheel(mp_obj_t n) {
-    return MP_OBJ_NEW_SMALL_INT(colorwheel(mp_obj_is_small_int(n) ? MP_OBJ_SMALL_INT_VALUE(n) : mp_obj_get_float(n)));
-}
-STATIC MP_DEFINE_CONST_FUN_OBJ_1(pixelbuf_colorwheel_obj, pixelbuf_colorwheel);
-
-const int32_t colorwheel(float pos) {
-    if (pos > 255) {
-        pos = pos - ((uint32_t)(pos / 256) * 256);
-    }
-    if (pos < 85) {
-        return (uint8_t)(255 - (pos * 3)) << 16 | (uint8_t)(pos * 3) << 8;
-    } else if (pos < 170) {
-        pos -= 85;
-        return (uint8_t)(255 - (pos * 3)) << 8 | (uint8_t)(pos * 3);
-    } else {
-        pos -= 170;
-        return (uint8_t)(pos * 3) << 16 | (uint8_t)(255 - (pos * 3));
-    }
-}
-
 STATIC const mp_rom_map_elem_t pixelbuf_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR___name__), MP_ROM_QSTR(MP_QSTR__pixelbuf) },
     { MP_ROM_QSTR(MP_QSTR_PixelBuf), MP_ROM_PTR(&pixelbuf_pixelbuf_type) },
-    { MP_ROM_QSTR(MP_QSTR_wheel), MP_ROM_PTR(&pixelbuf_colorwheel_obj) },
```

Comment:
I am assuming you probably did not want this keyword to continue on? I did not see it defined in the new module.

Reply:
You are correct! I did not want it to continue on.